examples/commoncrawl/build.gradle

```diff
@@ -0,0 +1,98 @@
+buildscript {
+	repositories {
+		mavenLocal()
+		maven { url "https://plugins.gradle.org/m2/" }
+		jcenter()
+	}
+	dependencies {
+		if (project.hasProperty("testing")) {
+			classpath "com.marklogic:ml-data-hub:5.4-SNAPSHOT"
+
+		} else {
+			classpath "com.marklogic:ml-data-hub:5.3.0"
+		}
+	}
+}
+plugins {
+	id "net.saliman.properties" version "1.5.1"
+}
+
+apply plugin: "com.marklogic.ml-data-hub"
+
+configurations {
+	mlcp
+}
+
+
+task clearStagingDatabase(type: com.marklogic.gradle.task.databases.ClearDatabaseTask) {
+  doFirst {
+    project.ext.database = project.properties.mlStagingDbName
+    project.ext.confirm = "true"
+  }
+  outputs.upToDateWhen { false }
+}
+
+task clearFinalDatabase(type: com.marklogic.gradle.task.databases.ClearDatabaseTask) {
+  doFirst {
+    project.ext.database = project.properties.mlFinalDbName
+    project.ext.confirm = "true"
+  }
+  outputs.upToDateWhen { false }
+}
+
+repositories {
+    jcenter()
+    maven { url "http://developer.marklogic.com/maven2/" }
+}
+dependencies {
+    mlcp "com.marklogic:mlcp:10.0.5"
+    mlcp files("lib")
+}
+
+task loadWebViaMlcp(type: com.marklogic.gradle.task.MlcpTask) {
+    description = "Load Commoncrawl web documents via MLCP"
+    classpath = configurations.mlcp
+    port = 8010
+    output_collections = "WebPage,load,input"
+    input_file_path = "/project/archive1/common/DHS/commoncrawl/0"
```

Comment:
It's better to copy sample data to the example project. This location is not accessible to everyone.